release.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+VERSION=$(git describe --abbrev=0 --tags)
+ARCH="${ARCH:-64}"
+LDC_FLAGS=("-flto=full")
+unameOut="$(uname -s)"
+case "$unameOut" in
+    Linux*) OS=linux; ;;
+    Darwin*) OS=osx; LDC_FLAGS+=("-macosx_version_min 10.7" "-lcrt1.o"); ;;
```

Comment:
```
ldc2: Unknown command line argument '-macosx_version_min'.  Try: 'ldc2 -help'
ldc2: Did you mean '-dwarf-version'?
ldc2: Unknown command line argument '-lcrt1.o'.  Try: 'ldc2 -help'
```

:/

Reply:
These are linker flags. You can check the DStep Dub file.